_i18n/it/get-started/what-is-monero.md

```diff
@@ -0,0 +1,120 @@
+{% include untranslated.html %}
+<div class="site-wrap">
+    <section class="container">
+            <div class="row">
+                <section class="container about-monero full col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col"><h2>Quello che devi sapere</h2></div>
+                        </div>
+                        <div class="row middle-xs info-block-row private">
+                            <div class="col-lg-7 col-md-6 col-sm-8 col-xs-12  why-text">
+                                <h3>Monero è la cryptomoneta di riferimento tra quelle focalizzate in transazioni private e resistenti alla censura.</h3>
```

Comment:
"Monero è una delle tante cryptomonete il cui obiettivo è quello di rendere le transazioni completamente private utilizzando algoritmi sicuri ed efficienti. A differenze di altre monete simili, Monero è l'unica ad implementare la privacy di default."

Reply:
@serhack some of these suggestions look kinda arbitrary and don't really reflect the original source, i think we should stick on the source text as much as possible and change that if necessary. So to keep the localization consistent. Also, in this case, your suggested review is too long for a header

Reply:
That's okay for me. I didn't see any warnings for limits.